TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -81,46 +82,21 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
             )
             return
         }
-
-        // Check if selected account supports trading equities
-        TradeItSDK.linkedBrokerManager.getAvailableBrokers(
-            onSuccess: { brokers in
-                guard let broker = brokers.first(
-                    where: { broker in
-                        return broker.shortName == self.order.linkedBrokerAccount?.linkedBroker?.brokerName
-                    }
-                ), broker.equityServices()?.supportsTrading == true else {
-                    self.alertManager.showAlertWithMessageOnly(
-                        onViewController: self,
-                        withTitle: "Unsupported Account",
-                        withMessage: "The selected account does not support trading stocks. Please choose another account.",
-                        withActionTitle: "OK",
-                        onAlertActionTapped: {
-                            self.delegate?.invalidAccountSelected(
-                                onTradingTicketViewController: self,
-                                withOrder: self.order
-                            )
-                        }
+        guard let _ = self.orderCapabilities else {
```

Comment:
This doesn't seem to be checking if equity trading is supported, just if orderCapabilities exists, which won't exist if we didn't call authenticate yet.

Reply:
This is set before. On `viewDidLoad` we call `updateOrderCapabilities` and in it: `self.orderCapabilities = (self.order.linkedBrokerAccount?.orderCapabilities.filter { $0.instrument == "equities" })?.first`
We set it also when we switch account.